src/EnergyPlus/ChillerGasAbsorption.cc

```diff
@@ -813,30 +747,28 @@ namespace ChillerGasAbsorption {
 			Node( GasAbsorber( ChillNum ).HeatSupplyNodeNum ).TempSetPointLo = Node( PlantLoop( GasAbsorber( ChillNum ).HWLoopNum ).TempSetPointNodeNum ).TempSetPointLo;
 		}
 
-		if ( ( GasAbsorber( ChillNum ).isWaterCooled ) && ( ( GasAbsorber( ChillNum ).InHeatingMode ) || ( GasAbsorber( ChillNum ).InCoolingMode ) ) && ( ! MyPlantScanFlag( ChillNum ) ) ) {
+		if ( ( GasAbsorber( ChillNum ).isWaterCooled ) && ( ( GasAbsorber( ChillNum ).InHeatingMode ) || ( GasAbsorber( ChillNum ).InCoolingMode ) ) && ( !Init_MyPlantScanFlag( ChillNum ) ) ) {
 			mdot = GasAbsorber( ChillNum ).DesCondMassFlowRate;
 			//DSU removed, this has to have been wrong (?)  Node(CondInletNode)%Temp  = GasAbsorber(ChillNum)%TempDesCondReturn
 
 			SetComponentFlowRate( mdot, GasAbsorber( ChillNum ).CondReturnNodeNum, GasAbsorber( ChillNum ).CondSupplyNodeNum, GasAbsorber( ChillNum ).CDLoopNum, GasAbsorber( ChillNum ).CDLoopSideNum, GasAbsorber( ChillNum ).CDBranchNum, GasAbsorber( ChillNum ).CDCompNum );
 
 		} else {
 			mdot = 0.0;
-			SetComponentFlowRate( mdot, GasAbsorber( ChillNum ).CondReturnNodeNum, GasAbsorber( ChillNum ).CondSupplyNodeNum, GasAbsorber( ChillNum ).CDLoopNum, GasAbsorber( ChillNum ).CDLoopSideNum, GasAbsorber( ChillNum ).CDBranchNum, GasAbsorber( ChillNum ).CDCompNum );
+			if ( GasAbsorber( ChillNum ).CDLoopNum > 0 ){
```

Comment:
No, I think it will go into that else for other reasons, like when it is off. Maybe the code needs an else if ( !  GasAbsorber( ChillNum ).isWaterCooled)  here for all air cooled cases.